app/src/components/skill/SkillTile.jsx

```diff
@@ -1,15 +1,46 @@
 import React, { useState } from "react";
-import { ChevronDownIcon, ChevronRightIcon, StarIcon, PencilSquareIcon } from "@heroicons/react/20/solid";
+import {
+  ChevronDownIcon,
+  ChevronRightIcon,
+  StarIcon,
+  PencilSquareIcon,
+  TrashIcon,
+} from "@heroicons/react/20/solid";
 import { useUserContext } from "src/contexts/UserContext";
 import CourseBadge from "./CourseBadge";
+import DeactivateSkillModal from "./hr/DeactivateSkillModal";
+import StopDeactivateSkillModal from "./hr/StopDeactivateSkillModal";
 
-export default function SkillTile({ skillId, skillName, skillDesc, courses, isActive }) {
+export default function SkillTile({
```

Comment:
As mentioned, I will merge this PR first as I think it's better to do the refactoring after [Charmaine's PR](https://github.com/is212g7t5/spm/pull/82) is merged, since we are both working on similar function (HR soft-deletes skills).